client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -187,11 +205,48 @@ const ProductLightbox: React.FC< Props > = ( {
 									/>
 								</div>
 							) }
-							<PaymentPlan
-								isMultiSiteIncompatible={ isMultiSiteIncompatible }
-								siteId={ siteId }
-								product={ product }
-							/>
+							{ shouldShowOptions && showVariantsWithoutRadioButtons ? (
+								<div>
+									<FormFieldset className="multiple-choice-question">
+										<FormLegend className="multiple-choice-question-title">
+											{ PRODUCT_OPTIONS_HEADER[ product?.productSlug ] }
+										</FormLegend>
+									</FormFieldset>
+									<div
+										onClick={ () => onChangeOption( JETPACK_SOCIAL_BASIC_PRODUCT?.productSlug ) }
+									>
+										<PaymentPlan
+											isMultiSiteIncompatible={ isMultiSiteIncompatible }
+											siteId={ siteId }
+											product={ JETPACK_SOCIAL_BASIC_PRODUCT as SelectorProduct }
+											showPlansOneBelowTheOther={ true }
+											isActive={
+												product?.productSlug === JETPACK_SOCIAL_BASIC_PRODUCT?.productSlug
+											}
+										/>
+									</div>
+									<p />
+									<div
+										onClick={ () => onChangeOption( JETPACK_SOCIAL_ADVANCED_PRODUCT?.productSlug ) }
+									>
+										<PaymentPlan
+											isMultiSiteIncompatible={ isMultiSiteIncompatible }
+											siteId={ siteId }
+											product={ JETPACK_SOCIAL_ADVANCED_PRODUCT as SelectorProduct }
+											showPlansOneBelowTheOther={ true }
+											isActive={
+												product?.productSlug === JETPACK_SOCIAL_ADVANCED_PRODUCT?.productSlug
+											}
+										/>
+									</div>
+								</div>
```

Comment:
Can we extract this to a separate component and then conditionally render it here?